modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/ckeditor5/editor.scss

```diff
@@ -106,3 +106,9 @@
 		}
 	}
 }
+
+.write-assistant__content--highlight {
```

Comment:
@igor-franca you can send a follow up for this, just rename to `.writing-assistant__content--highlight` here as well.

Thanks!